index.html

```diff
@@ -201,26 +213,26 @@ <h4 class="message">
           I'm always interested in hearing about new projects, so if you'd
           like to chat please get in touch.
         </h4>
-
         <div class="formme">
-          <form action="https://formspree.io/f/mvodvrrn" method="POST">
+          <form action="https://formspree.io/f/mvodvrrn" method="POST" id="formId">
             <ul id="form">
               <li>
                 <input type="text" id="name" maxlength="30" name="user_name" placeholder="Full name" required />
               </li>
               <li>
-                <input type="email" id="mail" name="user_email" placeholder="Email address"  required />
+                <input type="email" id="mail" name="user_email" placeholder="Email address" required />
               </li>
               <li>
                 <textarea maxlength="500" id="msg" name="user_message" placeholder="Enter text here" rows="5" cols="5"
                   required></textarea>
               </li>
 
               <li>
-                <div class="getintouch">
-                  <button class="btn-btn">Get in touch</button>
+                <!-- <div class="getintouch"> -->
+                  <button type="submit" id="contactme-btn" class="btn-btn">Get in touch</button>
```

Comment:
**Optional Changes**
- [ ] The submit button styling can be improved to resemble the figma design template.